src/com/google/javascript/jscomp/Es6ConvertSuper.java

```diff
@@ -108,44 +108,59 @@ private boolean isInterface(Node classNode) {
   }
 
   private void visitSuper(Node node, Node parent) {
-    Node enclosingCall = parent;
-    Node potentialCallee = node;
-    if (!parent.isCall()) {
-      enclosingCall = parent.getParent();
-      potentialCallee = parent;
+    Node exprRoot = node;
+    // properties with getElems aren't considered qualified names, so we have
+    // to do this without using any qName helpers.
+    for (;
+        exprRoot.getParent().isGetElem() || exprRoot.getParent().isGetProp();
+        exprRoot = exprRoot.getParent()) {
+      if (exprRoot.getParent().getFirstChild() != exprRoot) {
+        return;
+      }
     }
-    if (!enclosingCall.isCall()
-        || enclosingCall.getFirstChild() != potentialCallee
-        || enclosingCall.getFirstChild().isGetElem()) {
-      compiler.report(JSError.make(node, CANNOT_CONVERT_YET,
-          "Only calls to super or to a method of super are supported."));
-      return;
+
+    Node enclosingMemberDef =
+        NodeUtil.getEnclosingNode(
+            exprRoot,
+            new Predicate<Node>() {
+              @Override
+              public boolean apply(Node n) {
+                switch (n.getToken()) {
+                  case MEMBER_FUNCTION_DEF:
+                  case GETTER_DEF:
+                  case SETTER_DEF:
+                  case COMPUTED_PROP:
```

Comment:
I believe it was this test case: https://github.com/google/closure-compiler/pull/2418/files#diff-8cd1c911f8bd71b40937a5cdb2dbca85R1025